extensions/opentelemetry/opentelemetry/deployment/src/main/java/io/quarkus/opentelemetry/deployment/OpenTelemetryProcessor.java

```diff
@@ -54,7 +54,8 @@ public class OpenTelemetryProcessor {
     private static final DotName SPAN_ATTRIBUTE = DotName.createSimple(SpanAttribute.class.getName());
 
     static class RestClientAvailable implements BooleanSupplier {
-        private static final boolean IS_REST_CLIENT_AVAILABLE = isClassPresent("javax.ws.rs.client.ClientRequestFilter");
+        private static final boolean IS_REST_CLIENT_AVAILABLE = isClassPresent(
+                "io.quarkus.restclient.runtime.RestClientRecorder");
```

Comment:
Can we use the capabilities instead? This is extremely brittle. (Granted, it was already but it's even worse if you are using an implementation class)

Reply:
Yes, this was just a quick fix.

Reply:
Yep, I was about to say that capabilities are indeed the idiomatic way to do this stuff in quarkus :-)